### ISSUE TITLE
WolfHUD is crashing in a multiplayer lobby, if you aren't the host

### ISSUE THREAD

**Issue by kLrMicha** [Kamikaze94/WolfHUD#438]
As soon as you start a multiplayer heist and you are not the host, the game crashs right away. Following are the logs i'm getting, if you need any further information, just let me know about it.
Those issues are on the newest version of WolfHUD (updated 20mins ago).

BLT Log: 

```
11:27:59 PM Log:  (..\src\InitiateState.cpp:658) Initiating Hook
11:27:59 PM Lua: [BLT] Setup...
11:27:59 PM Lua: [BLT] Loading module: 	BLTLogs
11:27:59 PM Lua: [BLT] Loading module: 	BLTModManager
11:27:59 PM Lua: [BLT] Loading module: 	BLTKeybindsManager
11:27:59 PM Lua: [BLT] Loading module: 	BLTPersistScripts
11:27:59 PM Lua: [BLT] Loading module: 	BLTLocalization
11:27:59 PM Lua: [BLT] Cleaning logs folder, lifetime 1 day(s)
11:27:59 PM Lua: [BLT] Loading mods for state: table: 0x1dfa2900
11:27:59 PM Lua: [BLT] Loading mod: base
11:27:59 PM Lua: [BLT] Loading mod: WolfHUD
11:27:59 PM Lua: [BLT] Loading mod: _temp
11:27:59 PM Lua: [BLT] Could not read or find mod.txt in _temp
11:27:59 PM Lua: [BLT] Setting up mod: 	base
11:27:59 PM Lua: [BLT] Setting up mod: 	WolfHUD
11:28:00 PM FATAL ERROR:  (..\src\InitiateState.cpp:408) mods/WolfHUD/lua/GameInfoManager.lua:3802: attempt to index a nil value
stack traceback:
	mods/WolfHUD/lua/GameInfoManager.lua:3802: in main chunk
	[C]: in function 'dofile'
	mods/WolfHUD/Core.lua:1437: in main chunk
	[C]: in function 'dofile'
	mods/base/base.lua:120: in function 'RunHookFile'
	mods/base/base.lua:113: in function 'RunHookTable'
	mods/base/base.lua:142: in function 'require'
	[string "lib/player_actions/playeraction.lua"]:10: in main chunk
	[C]: in function 'require'
	mods/base/base.lua:141: in function 'require'
	[string "lib/setups/setup.lua"]:35: in main chunk
	...
	[string "lib/setups/gamesetup.lua"]:6: in main chunk
	[C]: in function 'require'
	mods/base/base.lua:141: in function 'require'
	[string "lib/setups/networkgamesetup.lua"]:1: in main chunk
	[C]: in function 'require'
	mods/base/base.lua:141: in function 'require'
	[string "lib/entry.lua"]:6: in main chunk
	[C]: in function 'require'
	mods/base/base.lua:141: in function 'require'
	[string "core/lib/coreentry.lua"]:19: in main chunk
11:28:01 PM Lua: [Menus][Warning] Parent menu for child 'wolfhud_main_options_menu' is null, ignoring...    1
11:28:09 PM Log:  (..\src\http\http.cpp:49) CURL CLOSED
```

Crashlog:

```
Application has crashed: access violation

-------------------------------

Callstack:

         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
                           ???  (???)     ???                                                 
                           ???  (???)     ???                                                 
                           ???  (???)     ???                                                 


-------------------------------

Current thread: Main
Script stack:
                     produce()  lib/managers/mission/elementspawnenemydummy.lua:83  
                 on_executed()  lib/managers/mission/elementspawnenemygroup.lua:81  
         execute_on_executed()  core/lib/managers/mission/coremissionscriptelement.lua:209  
_trigger_execute_on_executed()  core/lib/managers/mission/coremissionscriptelement.lua:147  
                 on_executed()  core/lib/managers/mission/coremissionscriptelement.lua:131  
                 on_executed()  lib/managers/mission/elementfilter.lua:34           
          client_on_executed()  lib/managers/mission/elementfilter.lua:10           
  client_run_mission_element()  lib/managers/missionmanager.lua:288                 
                                lib/network/handlers/unitnetworkhandler.lua:568     

-------------------------------

System information:
	Application version : 1.76.315.
	CPU : Intel(R) Core(TM) i5-3330 CPU @ 3.00GHz (2 cores); SSE; SSE2; SSE3; SSSE3; SSE4.1; SSE4.2
	DirectX : 12.0 
	GPU : NVIDIA GeForce GTX 970 / nvldumd.dll[22.21.13.8528]
	Language : @IDb6ce5cf32130deac@
	Memory :     12252MB 904KB
	OS : 6.2.9200 () 0x300-0x1 (64 bits)
	Physics : threaded
	Renderer : DX9 threaded
	Sound : Logitech (Lautsprecher (2- Logitech G930 Gaming Headset))
```

**Comment by FrostFire25699** [comment]
Even without mods I'm sure that happens to me too, with just BLT itself

**Comment by PhantomGamers** [comment]
This is a BLT issue, a temporary fix is to use the base folder from the 1.08 release of BLT with the other files from 2.0. The issue for BLT can be tracked here: https://github.com/JamesWilko/Payday-2-BLT/issues/89

**Comment by chromKa** [comment]
https://steamcommunity.com/app/218620/discussions/15/1474221865189786760/
